_includes/scenarios/integrate/read-attribute-from-peer.md

```diff
@@ -0,0 +1,249 @@
+
+- Accept sharing of existing [EMailAddress]({% link _docs_integrate/attribute-values.md %}#emailaddress):
+
+  | Property            | Value                                      |
+  | ------------------- | ------------------------------------------ |
+  | accept              | `true`                                     |
+  | existingAttributeId | `"<ID of Attribute of type EMailAddress>"` |
+
+- Reject sharing of [PhoneNumber]({% link _docs_integrate/attribute-values.md %}#phonenumber):
+
+  | Property | Value   |
+  | -------- | ------- |
+  | accept   | `false` |
+
+Note that it is important to respond to [RequestItems]({% link _docs_integrate/data-model-overview.md %}#requestitem) and [RequestItemGroups]({% link _docs_integrate/data-model-overview.md %}#requestitemgroup) in the same order in which they were received.
+
+## Get the Attributes
+
+We now assume that the Recipient has accepted the [Request for reading Attributes]({% link _docs_integrate/read-attribute-from-peer.md %}#request-for-reading-attributes) of the Sender. In order for the Sender to receive the response of the Recipient, it needs to [synchronize the updates of the Backbone]({% link _docs_use-cases/use-case-transport-synchronize-updates-of-backbone.md %}).
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/c8e5575e-ed55-48c2-897e-68e99d5df932" id="jbZRol4VVGZQ"></iframe></div>
+
+To view the response to the Request, search for it in the synchronization result or proceed as described in the [Get outgoing Request]({% link _docs_use-cases/use-case-consumption-get-outgoing-request.md %}) use case documentation and use the following query parameter:
+
+- If the [Request was sent over a Template]({% link _docs_integrate/read-attribute-from-peer.md %}#request-over-template): Specify `<ID of RelationshipTemplate>` as the value for the `source.reference` query parameter.
+- If the [Request was sent over a Message]({% link _docs_integrate/read-attribute-from-peer.md %}#request-over-message): Specify `<ID of Request>` as the value for the `id` query parameter.
+
+The Integrator of the Sender can now get the [Response]({% link _docs_integrate/data-model-overview.md %}#response) of the Recipient from the `response.content` property of the result. In particular, each requested and shared Attribute that belongs to an accepted ReadAttributeRequestItem can be read from a corresponding [AcceptResponseItem]({% link _docs_integrate/data-model-overview.md %}#acceptresponseitem) of type [ReadAttributeAcceptResponseItem]({% link _docs_integrate/requests-and-requestitems.md %}#readattributerequestitem-response-itemproperties) within the `items` property of the [Response]({% link _docs_integrate/data-model-overview.md %}#response):
```

Comment:
"The Integrator of the Sender" -> "The sender" (includes App Users)

Reply:
As in the currently published documentation (e.g. https://enmeshed.eu/integrate/requests-over-templates), I have now added a comment at the beginning that we are describing the reading process between two connectors for simplicity, but the general reading process between a connector and an app user would be the same.